### DISCUSSION TITLE
Possible to use Fictiverse's webui with BARK Infinity?

### DISCUSSION THREAD

**Discussion by Starfire70** [JonathanFly/bark#29] [Q&A]
I tried fiddling around with merging them but am getting an out of memory error, even though I have the line to use simple models in the .bat

Any help appreciated as using the command line is rather convoluted if you want to switch around voices, etc.

**Comment by JonathanFly** [comment]
We've both got webui's now. If I can get to it I will also use the [oobabooga](https://github.com/oobabooga/text-generation-webui) method for creating an installer.
